rover_msgs/AutonState.lcm

```diff
@@ -1,5 +0,0 @@
-package rover_msgs;
```

Comment:
why is this deleted?? jetson-teleop crashes now since auton state doesn't exist. is this file not supposed to be deleted or is jetson/teleop supposed to be changed?